odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -256,6 +256,12 @@ def test_compute_stored_many2one_one2many(self):
         # at this point, member.container_id must be computed for member to
         # appear in container.member_ids
         self.assertEqual(container.member_ids, member)
+        self.assertEqual(container.member_count, 1)
+
+        # indirectly detach member from container; this should recompute
+        # container.member_count
+        member.name = 'Bar'
+        self.assertEqual(container.member_count, 0)
```

Comment:
The comment could mention that modifying `member.name` should cause the recomputation of `member.container_id`, `container.member_ids` and `container.member_count`.
I suggest to add an extra step that does `member.name = 'Foo'` to check that we get back to the former situation, so with have both a modification that _detaches_ and another one that _attaches_.